apps/ledger-live-desktop/src/renderer/families/tron/operationDetails.tsx

```diff
@@ -102,13 +102,16 @@ export const OperationDetailsVotes = ({
 };
 
 const OperationDetailsExtra = ({
-  extra,
+  operation,
   type,
   account,
-}: OperationDetailsExtraProps<TronAccount>) => {
+}: OperationDetailsExtraProps<TronAccount, Operation>) => {
+  const amount = operation.extra.frozenAmount
```

Comment:
I would create a TronOperation of type

```typescript
type TronOperation = Omit<Operation, "extra> & { extra: { ...} }
```

And use it here instead of casting with `as`